Frontend/ui-common/src/Engineering/Helpers/Commons.js

```diff
@@ -459,3 +459,24 @@ export const getImageUrl = function (url) {
     console.log("error in getImageUrl " + e);
   }
 };
+
+export const getTimestampObject = function(string){
+  return function(){
+    if (!window.flowTimeStampObject || !(string in window.flowTimeStampObject)){
+      window.flowTimeStampObject[string] = Date.now() - window.prevTimeStamp;
```

Comment:
how will know the timestamp order ?

Reply:
Currently inside the flow first we'll have the baseappflow and further other flows, so later if we want some other timestamps we can name them accordingly